x-pack/plugins/apm/public/style/variables.ts

```diff
@@ -46,6 +46,10 @@ export const fontSizes = {
   xxlarge: '30px'
 };
 
+export const colors = {
```

Comment:
Would prefer to import the EUI colours by `import theme from '@elastic/eui/dist/eui_theme_light.json';` and use the `euiColorDanger` from the theme. It's already been tweaked slightly in EUI since I made the designs, so the HEX doesn't match.

PS: Shed a small tear when you added the colors back in our variables because I went through and replaced them all with EUI a few weeks back 😢 I'd prefer going forward that we try and use their theming going forward. I'll make sure to reference them in my issues too 👍